server/data/incidentClient.test.ts

```diff
@@ -59,9 +59,13 @@ test('getReportForReviewer', () => {
 test('getIncompleteReportsForReviewer', () => {
   const isOverdue = `(select count(*) from "v_statement" s
                       where r.id = s.report_id 
-                      and s.statement_status = $3
+                      and (s.statement_status = $3 or s.statement_status = $4)
```

Comment:
need to select where either OVERDUE or REMOVAL_REQUESTED

Reply:
this doesn't seem right? If status was removal requested then this wouldn't be overdue?

Reply:
hmm. Yes on second thoughts, it doesn't make sense. 
The overdue status is captured a few lines below already.